revisions/README_PacBio_CCS_15kb_20kb_HG002.md

```diff
@@ -1,5 +1,6 @@
 SAMPLE
     HG002 extracted DNA from Coriell Institute for Medical Research
```

Comment:
Change text to: "HG002 GIAB sample. Genomic DNA extracted from Coriell Institute for Medical Research, https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=NA24385&Product=DNA)"